README.md

```diff
@@ -0,0 +1,77 @@
+# Unikraft at Google Summer of Code 2022 (GSoC'22)
+
+Make your way into open source developement with operating systems, cloud, virtualization and low-level topics.
+Apply to [GSoC22](https://summerofcode.withgoogle.com/) with [Unikraft](https://summerofcode.withgoogle.com/programs/2022/organizations/unikraft).
+
+[Google Summer of Code 2022](https://summerofcode.withgoogle.com/) (GSoC'22), a global online program focused on bringing new contributors into open source software development.
+GSoC Contributors work with an open source organization, such as Unikraft, on a 12+ week programming project under the guidance of mentors.
+
+[Unikraft](https://unikraft.org/) Unikraft is a fast, secure and open-source Unikernel Development Kit.
+By tailoring the operating system, libraries and configuration to the particular needs of your application, it vastly reduces virtual machine and container image sizes to a few KBs, provides blazing performance, and drastically cuts down your software stack’s attack surface.
+
+## How to Apply[^1]
+
+* Connect to the community
+* Get accustomed to the Unikraft development ecosystem
+* Make a small contribution
+* Decide on a GSoC project
+* Fill out an application
+
+### Connect to the community
+
+First join our [Discord server](https://bit.ly/UnikraftDiscord).
+This is where all (text) discussion and (live video) meetings take place.
+You will find multiple channels, each for a particular topic of the Unikraft ecosystem.
+
+As a newcomer, start by presenting yourself (name, university / occupation, interest in Unikraft, current experience).
+It's best if you used a `Firstname Lastname` ID to make it easy for others to get in touch with you.
+
+Ask any questions, worry not about anything, we're happy to assist you in feeling at home in the Unikraft community.
+In particular, use the `gsoc22` and the `support` channels.
+
+### Get accustomed to the Unikraft development ecosystem
+
+Unikraft development happens on the [Unikaft GitHub organization](https://github.com/unikraft/).
+
+For starts, check the first 2 sessions in [Unikraft Summer of Code](https://usoc21.unikraft.org/docs/).
+These give you a very nice set of initial hands-on tasks to get you started with grabbing, configuring, building and running Unikraft.
+Optionally, look into sessions 3, 4 and 5 as well.
+
+### Make a small contribution
+
+The best way to prepare your application is to make small contributions.
+
+First off, browse the [Unikraft Contribution Guidelines](https://unikraft.org/docs/contributing/).
+This give you an overview of how we expect you to write your code, create commits and make contributions and GitHub pull requests (PRs).
```

Comment:
```suggestion
This will give you an overview of how we expect you to write your code, create commits and make contributions and GitHub pull requests (PRs).
```